tests/test_warm_reboot.py

```diff
@@ -234,7 +234,94 @@ def ping_new_ips(dvs):
             dvs.runcmd(['sh', '-c', "ping -c 1 -W 0 -q {}.0.0.{} > /dev/null 2>&1".format(i*4,j+NUM_NEIGH_PER_INTF+2)])
             dvs.runcmd(['sh', '-c', "ping6 -c 1 -W 0 -q {}00::{} > /dev/null 2>&1".format(i*4,j+NUM_NEIGH_PER_INTF+2)])
 
+def create_mirror_session(dvs, name, src, dst, gre, dscp, ttl, queue):
+    tbl = swsscommon.Table(dvs.cdb, "MIRROR_SESSION")
+    fvs = swsscommon.FieldValuePairs([("src_ip", src),
+                                      ("dst_ip", dst),
+                                      ("gre_type", gre),
+                                      ("dscp", dscp),
+                                      ("ttl", ttl),
+                                      ("queue", queue)])
+    tbl.set(name, fvs)
+    time.sleep(1)
+
+def remove_mirror_session(dvs, name):
+    tbl = swsscommon.Table(dvs.cdb, "MIRROR_SESSION")
+    tbl._del(name)
+    time.sleep(1)
+
+def get_mirror_session_state(dvs, name):
```

Comment:
Just a suggestion, it might be good to leave a short comment describing the return value here. It makes sense to me because I've been working with the swsscommon library but to someone skimming over these tests it might not be super intuitive what's going on here.